PokeAlarm/Filters/EggFilter.py

```diff
@@ -21,6 +21,16 @@ def __init__(self, name, data):
             event_attribute='distance', eval_func=operator.ge,
             limit=BaseFilter.parse_as_type(float, 'max_dist', data))
 
+        # Time Left
+        self.min_time_to_hatch = self.evaluate_attribute(
+            # f.min_time_to_hatch <= r.hatch_time_left
+            event_attribute='hatch_time_left', eval_func=operator.le,
+            limit=BaseFilter.parse_as_type(int, 'min_time_to_hatch', data))
```

Comment:
The rest are all `min_time_left` and `max_time_left` - maybe this one should be as well?

Reply:
I thought about that when I was making it, was waiting for someone else to make the same suggestion. :)